pkg/config/loader_test.go

```diff
@@ -0,0 +1,330 @@
+package config
+
+import (
+	"os"
+	"reflect"
+	"testing"
+	"time"
+
+	autoheal "github.com/openshift/autoheal/pkg/apis/autoheal/v1alpha2"
+	batch "k8s.io/api/batch/v1"
+	core "k8s.io/api/core/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestFiles(t *testing.T) {
+	l := NewLoader()
+
+	filename0 := "test_file0.yaml"
+	filename1 := "test_file1.yaml"
+
+	data0 := `
+  awx:
+    address: "http://test_address.com"`
+
+	data1 := `
+  awx:
+    proxy: "http://test-proxy.com:1234"`
+
+	file0, _ := os.Create(filename0)
+	file1, _ := os.Create(filename1)
```

Comment:
Can we create these files in a temporary directory? Maybe using `ioutil.TempDir` or `ioutil.TempFile`. Otherwise, if we create them in the project directory, chances are that someone will add them to the repository by accident.